test/WebSites/BasicWebSite/Controllers/HomeController.cs

```diff
@@ -35,11 +35,21 @@ public IActionResult RedirectToActionReturningTaskAction()
             return RedirectToAction("ActionReturningTask");
         }
 
+        public IActionResult RedirectToActionWithFragment()
```

Comment:
Please revert the changes to this test controller. None of the functional tests uses the two new methods and I've decided coverage is fine with the new unit tests.